schema/me/conversation/index.js

```diff
@@ -190,7 +190,28 @@ export const ConversationFields = {
   last_message_open: {
     type: GraphQLString,
     description: "Timestamp if the user opened the last message, null in all other cases",
-    resolve: conversation => null, // eslint-disable-line no-unused-vars
+    resolve: (conversation, options, request, { rootValue: { accessToken } }) => {
+      const radiationMessageId = get(conversation, "_embedded.last_message.radiation_message_id")
+      const impulseParams = {
+        conversation_id: conversation.id,
+        radiation_message_id: radiationMessageId,
+        "expand[]": "deliveries",
+      }
+      return gravity.with(accessToken, { method: "POST" })("me/token", {
+        client_application_id: IMPULSE_APPLICATION_ID,
+      }).then(data => {
+        return impulse.with(data.token, { method: "GET" })(
+          `message_details`,
+          impulseParams
+        ).then(({ message_details }) => {
+          if (message_details.length === 0) {
+            return null
+          }
+          const relevantDelivery = message_details[0].deliveries.find(d => d.email === conversation.from_email)
```

Comment:
It's clear to me, generally speaking, how we might cache and transparently refresh these special tokens. That can help all uses of this grav immediately followed by an impulse call, of which there are several. 

Loading each conversations last message one a time is cool, but we can add an endpoint to load via id and then refactor the loader here to truly batch load in bulk. 

Etc...

So I think next rounds are pretty imminently, I believe we can merge this, iterate the next round, and so forth.